doc/man/meson.build

```diff
@@ -0,0 +1,3 @@
+install_man('mididings.1', install_dir: get_option('mandir'))
+install_man('livedings.1', install_dir: get_option('mandir'))
+install_man('send_midi.1', install_dir: get_option('mandir'))
```

Comment:
You don't need install_dir here, since the main purpose of install_man instead of install_data is that it handles mandir automatically, for you.

Reply:
Thanks for having a closer look! These lines are in fact remains of previous experiments. I'm reworking that part at the moment which includes generating the man pages with scdoc.